integrationtests/tests/fixtures/mod.rs

```diff
@@ -82,7 +82,8 @@ mod fake;
 mod real;
 mod utils;
 
-static BASE_PORT: AtomicU16 = AtomicU16::new(DEFAULT_P2P_PORT + 10000);
+const BASE_PORT_INIT: u16 = DEFAULT_P2P_PORT + 10000;
+static BASE_PORT: AtomicU16 = AtomicU16::new(BASE_PORT_INIT);
```

Comment:
Unrelated to this PR: The naming is a bit weird. Why does a **base** port change (it does I assume since it's an atomic int)?

Reply:
Ah, so it's a base for each run I guess. Makes sense.